32blit-sdl/UserCode.hpp

```diff
@@ -11,3 +11,6 @@ extern const char *metadata_description;
 extern const char *metadata_version;
 extern const char *metadata_url;
 extern const char *metadata_category;
+
+/* Added a command line ability to specify a launch_path parameter. */
+extern const char *launch_path;
```

Comment:
Hmm, doesn't feel like the right place for this... everything declared else declared here is defined in the "game".

Reply:
I sort of agree, but I couldn't work out a less "wrong" header file to put it in. I suppose I could have just extern'd it in System.cpp but that would have felt wrong too.

Reply:
But `System.cpp` *includes* `UserCode.hpp` so it's just one level of wrongness indirection :laughing: I'd toss it in `System.cpp` just so fewer files are touched by the change.

Reply:
Consider it done ;)